### PR TITLE
Minor improvements & Update help sections of --with-choreo flag

### PR DESCRIPTION
**Pull request by nadundesilva**
## Approach
* author javadoc tag which had been added mistakenly was removed
* since javadoc tag had which was missing in previous PRs were added (the updated since tags were tags added mistakenly while copying a class from another part of the repo itself)
* CLI help sections for `--with-choreo` flag had been added

### CODE REVIEW

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java [hunk 1]
```diff
@@ -181,7 +181,8 @@ public BuildCommand(Path userDir, PrintStream outStream, PrintStream errStream,
     @CommandLine.Option(names = "--code-coverage", description = "enable code coverage")
     private boolean coverage;
 
-    @CommandLine.Option(names = "--with-choreo", description = "package Choreo extension in the executable jar")
+    @CommandLine.Option(names = "--with-choreo", description = "package Choreo observability extension " +
```
Comment (MaryamZi) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java:184]: Shouldn't this and everywhere else be

```suggestion
    @CommandLine.Option(names = "--with-choreo", description = "package the Choreo observability extension " +
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help [hunk 2]
```diff
@@ -65,6 +65,9 @@ OPTIONS
        --debug
            Run tests in remote debugging mode, only when the '--skip-tests' flag is not used.
 
+       --with-choreo
+           Package Choreo observability extension in the executable jars.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help:69]: Wondering if we should just say

```suggestion
           Package the Choreo observability extension in the executables.
```

or

```suggestion
           Package the Choreo observability extension in the executable JAR file(s).
```

for consistency.

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help [hunk 3]
```diff
@@ -43,6 +43,9 @@ OPTIONS
        --debug
            Run in remote debugging mode.
 
+       --with-choreo
+           Package Choreo observability extension in the executable jars created while building.
```
Comment (MaryamZi) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help:47]: This wouldn't apply when `ballerina run` is used with a JAR right?

Should we say something like

```suggestion
           Package the Choreo observability extension in the executable when run is used with a source file or a module.
```

Comment (MaryamZi) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help:47]: Please check the line length too. IIRC, the maximum we used to have was 80.

Comment (nadundesilva) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help:47]: Yes these flags are applicable only when the commands are run against bal files.

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java (hunk 1 above)
Comment (praneesha) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java:184]: +1

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help (hunk 2 above)
Comment (praneesha) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help:69]: +1 for the 2nd suggestion.

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help (hunk 3 above)
Comment (praneesha) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help:47]: +1

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help [hunk 4]
```diff
@@ -65,6 +65,9 @@ OPTIONS
        --debug
            Run tests in remote debugging mode, only when the '--skip-tests' flag is not used.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help:66]:
```suggestion
           Run tests in the remote debugging mode only when the '--skip-tests' flag is not used.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help [hunk 5]
```diff
@@ -65,6 +65,9 @@ OPTIONS
...
 
+       --with-choreo
+           Package Choreo observability extension in the executable jars.
+
 CONFIG PROPERTIES
        (--key=value)...
            Set Ballerina environment parameters as key/value pairs.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help:73]:
```suggestion
           Set the Ballerina environment parameters as key/value pairs.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help [hunk 6]
```diff
@@ -43,6 +43,9 @@ OPTIONS
        --debug
            Run in remote debugging mode.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help:44]:
```suggestion
           Run in remote the debugging mode.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help [hunk 7]
```diff
@@ -43,6 +43,9 @@ OPTIONS
...
 
+       --with-choreo
+           Package Choreo observability extension in the executable jars created while building.
+
 CONFIG PROPERTIES
        (--key=value)...
            Set Ballerina environment parameters as key/value pairs.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help:51]:
```suggestion
           Set the Ballerina environment parameters as key/value pairs.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 8]
```diff
@@ -76,6 +76,9 @@ OPTIONS
        --debug
            Run tests in remote debugging mode.
 
+       --with-choreo
+           Package Choreo observability extension in the executable jars created while building.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help:80]:
```suggestion
           Package the Choreo observability extension in the executable JAR files created while building.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 9]
```diff
@@ -76,6 +76,9 @@ OPTIONS
...
+
 CONFIG PROPERTIES
        (--key=value)...
            Set Ballerina environment parameters as key/value pairs.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help:84]:
```suggestion
           Set the Ballerina environment parameters as key/value pairs.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help (hunk 6 above)
Comment (MaryamZi) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help:44]: I'm not sure if this change is correct. "remote debugging" is a thing.

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help (hunk 4 above)
Comment (MaryamZi) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help:66]: This too, shall we revisit and verify the best (common) terms to use, and fix in a separate PR maybe?

Comment (MaryamZi) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help:66]: I actually think we should use "remote debug mode", but let's verify and change this properly. @nadundesilva can you create an issue to track these changes please?

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java (hunk 1 above)
Comment (nadundesilva) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java]: Fixed in 77fd6022bdbaf2870da71ae578be26ded14c77b5

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help (hunk 2 above)
Comment (nadundesilva) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help]: Fixed in 77fd6022bdbaf2870da71ae578be26ded14c77b5

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help (hunk 3 above)
Comment (nadundesilva) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help]: Fixed in 77fd6022bdbaf2870da71ae578be26ded14c77b5

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help (hunk 5 above)
Comment (nadundesilva) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help]: Fixed in 77fd6022bdbaf2870da71ae578be26ded14c77b5

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help (hunk 7 above)
Comment (nadundesilva) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help]: Fixed in 77fd6022bdbaf2870da71ae578be26ded14c77b5

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help (hunk 8 above)
Comment (nadundesilva) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help]: Fixed in 77fd6022bdbaf2870da71ae578be26ded14c77b5

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help (hunk 4 above)
Comment (nadundesilva) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help:66]: Created Issue #24275

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help (hunk 6 above)
Comment (nadundesilva) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help:44]: Created Issue #24275

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help (hunk 4 above)
Comment (NipunaRanasinghe) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help:66]: > I actually think we should use "remote debug mode", but let's verify and change this properly. @nadundesilva can you create an issue to track these changes please?

True. +1 for using either of the terms ("remote debugging" or "remote debug"), but for the above context, I also prefer using **remote debug mode**.

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help (hunk 6 above)
Comment (NipunaRanasinghe) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help:44]: > I'm not sure if this change is correct. "remote debugging" is a thing.

Yeah I also think that this should better be kept as it is. +1 for the existing way.

**Pull request merged**